doc/source/indexing.rst

```diff
@@ -1525,20 +1525,35 @@ faster, and allows one to index *both* axes if so desired.
 Why does the assignment when using chained indexing fail!
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-So, why does this show the ``SettingWithCopy`` warning / and possibly not work when you do chained indexing and assignment:
+The above is just a performance issue. What's up with the ``SettingWithCopy`` warning? We don't **usually** throw warnings around when you do something that might cost a few extra milliseconds!
+
+But it turns out that assigning to the product of chained indexing has inherently unpredictable results. To see this, think about how the Python interpreter executes this code:
 
 .. code-block:: python
 
+   dfmi.loc[:,('one','second')] = value
+   # becomes
```

Comment:
I think you should write these as separate code-blocks (which you can then intersperce text)